plugins/modules/ssm_parameter.py

```diff
@@ -444,6 +572,8 @@ def setup_module_object():
         key_id=dict(default="alias/aws/ssm"),
         overwrite_value=dict(default='changed', choices=['never', 'changed', 'always']),
         tier=dict(default='Standard', choices=['Standard', 'Advanced', 'Intelligent-Tiering']),
+        tags=dict(type='dict', aliases=['resource_tags']),
+        purge_tags=dict(type='bool', default=True),
```

Comment:
Imo this is a breaking change. Because until `community.aws 5.1.0` you can use `ssm_parameter` and it won't touch the tags. But now it would just purge your tags.  
This must be set to `default=False` for the moment of release.  

cc @alinabuzachis

Reply:
Even with purge_tags=True as the default, existing tags will only be removed if the tags param is populated. To remove all tags, the empty dict needs to be supplied. I don't think this is a breaking change because someone would have had to set the tags parameter in existing tasks, which I believe would have thrown an error prior to this change.

Reply:
Ah right. thanks!